src/quest_manager/templates/quest_manager/category_detail.html

```diff
@@ -0,0 +1,11 @@
+{% extends "quest_manager/base.html" %}
+{% load crispy_forms_tags %}
+
+{% block heading_inner %} {{ heading }}{% endblock %}
+
+{% block content %}
+    {% with c as category %}
```

Comment:
use `with object as category` and you can remove the extra context data and method from the view